### ISSUE TITLE
Librespot Binaries

### ISSUE THREAD

**Issue by gwilherm55** [librespot-org/librespot#275]
For guy who don't want to build the source, they can use this: [https://github.com/gwilherm55/librespot_binaries](https://github.com/gwilherm55/librespot_binariesl)

**Comment by ashthespy** [comment]
Your link is broken - points to `https://github.com/gwilherm55/librespot_binariesl`

**Comment by gwilherm55** [comment]
[https://github.com/gwilherm55/librespot_binaries](https://github.com/gwilherm55/librespot_binaries)

**Comment by gwilherm55** [comment]
Thank you 👌

**Comment by kingosticks** [comment]
You should probably specify which backends are compiled and and the fact that you don't have the raspberry pi 1/zero build.

**Comment by gwilherm55** [comment]
`arm-unknown-linux-gnueabi ` its pi 1?

**Comment by kingosticks** [comment]
Nope. https://github.com/librespot-org/librespot/blob/master/contrib/docker-build-pi-armv6hf.sh

**Comment by gwilherm55** [comment]
I run this to have all the binaries : `docker run -v /tmp/librespot-build:/build librespot-cross
`

**Comment by gwilherm55** [comment]
When i use : `cargo build --release --target arm-unknown-linux-gnueabihf --no-default-features --features "alsa-backend" --verbose`
I got :```

error[E0463]: can't find crate for `std`
  |
  = note: the `arm-unknown-linux-gnueabihf` target may not be installed

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: Could not compile `libc`.

Caused by:
  process didn't exit successfully: `rustc --crate-name libc /root/.cargo/registry/src/github.com-1ecc6299db9ec823/libc-0.2.40/src/lib.rs --color always --crate-type lib --emit=dep-info,link -C opt-level=3 --cfg 'feature="default"' --cfg 'feature="use_std"' -C metadata=dff7834e8e33583f -C extra-filename=-dff7834e8e33583f --out-dir /root/librespot/target/arm-unknown-linux-gnueabihf/release/deps --target arm-unknown-linux-gnueabihf -L dependency=/root/librespot/target/arm-unknown-linux-gnueabihf/release/deps -L dependency=/root/librespot/target/release/deps --cap-lints allow` (exit code: 1)
```

**Comment by kingosticks** [comment]
Yeh sure, I can see what you've done but I'm just saying that isn't *all* the binaries. If you also want to provide the single armhf binary that works on all raspberry pi models then you must also run the other docker image. If you look you'll see it's different. 

If you don't, that's fine of course, but you should make it clear as it will confuse some people. And then perhaps instead point to whatever that debian packaged thing is called.

**Comment by gwilherm55** [comment]
I updated the README

**Comment by kingosticks** [comment]
Have you tried that 2nd binary on a raspbery pi 1?

**Comment by sashahilton00** [comment]
We'll start doing builds properly at some point, likely once we start versioning things properly and publish to crates.io. There are still a few things which we need to fix before I think we're ready to publish, those being handling reconnections, gapless, and daily mixes. The other issues can be fixed in later versions, as they're not game changing.

**Comment by gwilherm55** [comment]
I dont know how i can build the 2nd binary on raspberry pi 1

**Comment by gwilherm55** [comment]
If you need to change something in my repository make a pull request

**Comment by kingosticks** [comment]
I think this can be closed unless we need an example to remind why the wiki needs updating.
